src/store/DevfileRegistries/index.ts

```diff
@@ -171,11 +171,15 @@ export const actionCreators: ActionCreators = {
         const patchedJSONString = JSON.stringify(schemav1).replaceAll('#/definitions', '#/oneOf/0/definitions');
         const parsedSchemaV1 = JSON.parse(patchedJSONString);
 
-        const schemav2 = await WorkspaceClient.restApiClient.getDevfileSchema('2.0.0');
+        const schemav200 = await WorkspaceClient.restApiClient.getDevfileSchema('2.0.0');
+        const schemav210 = await WorkspaceClient.restApiClient.getDevfileSchema('2.1.0');
```

Comment:
I wonder if devfile schema should set an explicit value for schema version, like here https://github.com/devfile/api/blob/main/schemas/latest/devfile.json#L1560
It would guarantee that devfile if validate with a proper version of the schema.
I'll raise it on the Devfile Cabal